sigmastate/src/main/scala/sigmastate/interpreter/CryptoFunctions.scala

```diff
@@ -5,8 +5,12 @@ import scorex.crypto.hash.Blake2b256
 object CryptoFunctions {
   lazy val soundnessBytes: Int = CryptoConstants.soundnessBits / 8
 
+  /** @hotspot don't beautify this code. Used in Interpreter.verify. */
```

Comment:
Please add method desc, please note that it is not so clear now from the code that trimming takes place.

Reply:
added detailed description